google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -125,6 +125,16 @@ class SpannerSQLCompiler(SQLCompiler):
 
     compound_keywords = _compound_keywords
 
+    def get_from_hint_text(self, _, text):
+        """Return a hint text.
+
+        Overriden to avoid adding square brackets to the hint text.
+
+        Args:
+            text (str): The hint text.
+        """
+        return text
+
```

Comment:
The original method does the same, but also adds square brackets without conditions and explanations